.github/workflows/docker.yml

```diff
@@ -62,10 +62,12 @@ jobs:
             make install
           binaries: |
             - /go/bin/provenanced
+# override WASMVM_REPO since github.com/CosmWasm/wasmvs/v2 path doesn't exist in github
```

Comment:
While just a comment the spelling issue here: `wasmvs/v2` bugs me

Reply:
:cry: it actually wasn't a variable for the command so it's been reverted anyways